src/System.Diagnostics.Process/tests/ProcessTests.netstandard1.7.cs

```diff
@@ -25,6 +25,40 @@ public void TestHandleCount()
             }
         }
 
+        [Fact]
+        [PlatformSpecific(TestPlatforms.OSX)]
+        public void TestHandleCount_OSX()
+        {
+            using (Process p = Process.GetCurrentProcess())
+            {
+                Assert.Equal(0, p.HandleCount);
+            }
+        }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Linux | TestPlatforms.Windows)]
+        public void HandleCountChanges()
+        {
+            RemoteInvoke(() =>
+            {
+                Process p = Process.GetCurrentProcess();
+                int firstHandleCount = p.HandleCount;
+                var fs1 = File.Open(Path.Combine(Path.GetTempPath(), Path.GetTempFileName()), FileMode.OpenOrCreate);
+                var fs2 = File.Open(Path.Combine(Path.GetTempPath(), Path.GetTempFileName()), FileMode.OpenOrCreate);
+                var fs3 = File.Open(Path.Combine(Path.GetTempPath(), Path.GetTempFileName()), FileMode.OpenOrCreate);
+                p.Refresh();
+                int secondHandleCount = p.HandleCount;
+                Assert.True(firstHandleCount < secondHandleCount);
```

Comment:
How do you know this will never fail due to other unrelated handles being released concurrently by unrelated activity on the box?

perhaps if you opened 1000's of files..